tools/configure/configure.ml

```diff
@@ -357,8 +353,8 @@ let args_options = Arg.align [
     " Build bytecode executables with -custom (not recommended)";
   "-no-custom", arg_clear_option (fun p custom -> { p with custom }),
     " Do not build with -custom on Windows and MacOS";
-  "-bindir", arg_string_option (fun p bindir -> { p with bindir }),
-    "<dir> Where to install bin files";
+  "-bindir", arg_string_option (fun p _ -> p ),
+    "deprecated option, use -prefix";
```

Comment:
Can you make it clearer that this is not only deprecated but also a no-op?

And could you trigger an actual warning when this is used, similar to what was done for `local`?